app/scripts.babel/bkg/designer-live-preview.js

```diff
@@ -120,9 +139,10 @@ const enable = (projectName, nuxeoInstanceBaseUrl) => {
     urls: [`${CONNECT_URL}/nuxeo/site/studio/v2/project/${projectName}/workspace/ws.resources`],
   }, ['requestBody']);
 
+  // https://groups.google.com/a/chromium.org/g/chromium-extensions/c/vYIaeezZwfQ
   browser.webRequest.onBeforeSendHeaders.addListener(addCookieHeaderForConnectRequest, {
     urls: [`${CONNECT_URL}/*`],
-  }, ['blocking', 'requestHeaders']);
+  }, ['blocking', 'requestHeaders', 'extraHeaders']);
```

Comment:
Maybe you should do the check described here https://groups.google.com/a/chromium.org/g/chromium-extensions/c/vYIaeezZwfQ/m/hVln6g1OAgAJ for versions of chrome before 72?